clang/include/clang/Basic/Attr.td

```diff
@@ -1560,6 +1560,41 @@ def IntelFPGAMaxPrivateCopies : InheritableAttr {
   }];
 }
 
+// Two string arguments.
+def IntelFPGAMerge : Attr {
+  let Spellings = [CXX11<"intelfpga","merge">];
+  let Args = [StringArgument<"Name">, StringArgument<"Direction">];
+  let Subjects = SubjectList<[IntelFPGAConstVar, IntelFPGALocalOrStaticVar,
+                              Field], ErrorDiag>;
+  let LangOpts = [SYCL];
+  let Documentation = [Undocumented];
```

Comment:
Is anywhere spec for the feature? If it's undocumented attribute I would like to see some link at least in the commit message.